doc/whats_new/v1.1.rst

```diff
@@ -462,6 +470,7 @@ Changelog
   left corner of the HTML representation to show how the elements are
   clickable. :pr:`21298` by `Thomas Fan`_.
 
+
```

Comment:
This change is unrelated. Can you revert?

Reply:
@thomasjpfan This line belongs to the enhancement changelog entry which was not part of this PR changes:
```
- |Enhancement| :func:`utils.estimator_html_repr` displays an arrow on the top
  left corner of the HTML representation to show how the elements are
  clickable. :pr:`21298` by `Thomas Fan`_.
```

Reply:
There is a blank line here

Reply:
Maybe it's the GitHub interface. I am referring to this newline added by this PR:

![Screen Shot 2022-01-17 at 9 02 06 AM](https://user-images.githubusercontent.com/5402633/149782516-cf6835f7-b48f-43e2-b9e5-caaa1f7f3958.jpg)

Reply:
Thanks for the clarification.